### DISCUSSION TITLE
Client doesn't start

### DISCUSSION THREAD

**Discussion by BudliCode** [ORelio/Minecraft-Console-Client#2139] [General]
Hi There.
When i try to start the client nothing happens.
When trying to open in cmd i got this result:

C:\Users\xyzr>C:\Users\xyz\Desktop\MinecraftClient-windows\MinecraftClient.exe
Failure processing application bundle.
Bundle header version compatibility check failed.
A fatal error occured while processing application bundle

im using the latest build (20220830-35) on Windows.
Any Ideas how to fix this?

**Comment by BudliCode** [comment]
older versions don't seem to work as well. might be a problem on my pc

**Comment by breadbyte** [comment]
Please try installing or reinstalling [.NET 6.0 Runtime](https://dotnet.microsoft.com/en-us/download).

**Reply by BudliCode** [reply]
Installed and it runs like a charm. 
Thanks a lot!
